geopmdpy/test/TestAccess.py

```diff
@@ -378,6 +379,22 @@ def test_read_log_controls(self):
         expected_result = '\n'.join(self._controls_expect)
         self.assertEqual(expected_result, actual_result)
 
+    def test_group_id(self):
+        """Test that CLI group name conversion works
+
+        """
+        group_id = 101
+        class MockGrp:
+             def __init__(self):
+                 self.gr_name = 'group_name'
+                 self.gr_passwd = 'x'
+                 self.gr_gid = group_id
+                 self.gr_mem = []
+        expected = str(group_id)
+        ret = MockGrp()
+        self.assertEqual(expected, _group_id(expected))
+        with mock.patch('grp.getgrnam', return_value=ret):
+            self.assertEqual(expected, _group_id('group_name'))
```

Comment:
Sorry, the slightest hint of duplicate code bugs me. Feel free to disregard.

```suggestion
        group_id = 101
        group_name = 'group_name'
        class MockGrp:
             def __init__(self):
                 self.gr_name = group_name
                 self.gr_passwd = 'x'
                 self.gr_gid = group_id
                 self.gr_mem = []
        expected = str(group_id)
        ret = MockGrp()
        self.assertEqual(expected, _group_id(expected))
        with mock.patch('grp.getgrnam', return_value=ret):
            self.assertEqual(expected, _group_id(group_name))
```